model/crystal.h

```diff
@@ -461,8 +468,13 @@ namespace dxtbx { namespace model {
     /**
      * @returns the unit_cell
      */
-    cctbx::uctbx::unit_cell get_unit_cell() const {
-      return unit_cell_;
+    cctbx::uctbx::unit_cell get_unit_cell(bool postrefined=false) const {
+      if (postrefined){
+        DXTBX_ASSERT(postrefined_cell_is_set_);
```

Comment:
is a `ValueError` better than a `DIALS_ASSERT`?

Reply:
Can I raise that from C++? Otherwise perhaps could catch this on the Python side and convert to `ValueError`